dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic

```diff
@@ -131,6 +131,8 @@ NoNetworkConnection           =             component %1% not imported : no netw
 PossibleDivisionByZero        =             impedance of line ID: %1% set to 0 in IIDM file, default parameters are then used: R = 0 and X = 0.01
 CriteriaDefinedButNoIIDM      =             a criteria file was defined without any IIDM file, it will be ignored
 //---------------------- OTHERS --------------------------------------
+// --> ModelAreaShedding
+LoadSheddingValueIncomplete   =             load shedding quantities are not fully provided in the AreaShedding model's paramters, use values in the timeline with cautious
```

Comment:
I would propose " load shedding quantities are not fully provided in model %1%'s parameters, using default (0.) for missing values"